src/lib/storage/vector_compression/bitpacking/bitpacking_decompressor.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "bitpacking_vector_type.hpp"
+#include "compact_vector.hpp"
+#include "storage/vector_compression/base_vector_decompressor.hpp"
+
+namespace opossum {
+
+class BitPackingVector;
+
+class BitPackingDecompressor : public BaseVectorDecompressor {
+ public:
+  explicit BitPackingDecompressor(const pmr_compact_vector<uint32_t>& data) : _data{data} {}
+  BitPackingDecompressor(const BitPackingDecompressor& other) = default;
+  BitPackingDecompressor(BitPackingDecompressor&& other) = default;
+
+  BitPackingDecompressor& operator=(const BitPackingDecompressor& other) {
+    DebugAssert(&_data == &other._data, "Cannot reassign BitPackingDecompressor");
```

Comment:
What cases do we catch here? Why don't we delete the copy constructor?

Reply:
I think I've found the reason and I might have a look at it at a later point in time. Not your issue.

Reply:
just copied this from the other decompressors

Reply:
Yeah, not your problem. The reason has been the state in the SimdBp128 decompressor. As we now do no longer have any state in the decompressors, we can easily pass references. 
But I am not sure if makes a measurable difference. Anyhow, not your problem.